mmv1/products/bigquerydatatransfer/terraform.yaml

```diff
@@ -37,6 +37,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           to a different credential configuration in the config will require an apply to update state.
       sensitiveParams.secretAccessKey: !ruby/object:Overrides::Terraform::PropertyOverride
         sensitive: true
+      # api does not send schedule_options.disable_auto_scheduling back when it is set to false b/262259565
+      scheduleOptions: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: 'emptyOrUnsetBlockDiffSuppress'
```

Comment:
Can this be handle by setting optional + compute

Reply:
I don't think so. In this case, api sends empty. Can Computed work for thing like that?

Reply:
Update: a custom diff suppress func will be added to replace `emptyOrUnsetBlockDiffSuppress` as this should be an easy logic and plus, we recommend against using `emptyOrUnsetBlockDiffSuppress`.

Reply:
@shuyama1 does the new function available or can we merge this PR with ```emptyOrUnsetBlockDiffSuppress ```?

Reply:
@edwardmedia gentle ping. Any update on this?